controllers/cards.js

```diff
@@ -1,36 +1,43 @@
+const mongoose = require('mongoose');
 const Card = require('../models/card');
 
-module.exports.getAllCards = (req, res) => {
+const { ObjectId } = mongoose.Types;
+
+const NotFoundError = require('../errors/error_not_found');
+const Error403 = require('../errors/error_403');
+
+module.exports.getAllCards = (req, res, next) => {
   Card.find({})
-    .then((card) => {
-      if (card.length === 0) {
-        return res.status(404).send({ message: 'База данных cards пуста! ' });
-      }
-      return res.send({ data: card });
-    })
-    .catch((error) => res.status(500).send({ message: error.message }));
+    .then((card) => res.send({ data: card }))
+    .catch(next);
 };
 
-module.exports.createCard = (req, res) => {
-  const owner = req.user._id;
+module.exports.createCard = (req, res, next) => {
+  /* const owner = req.user._id; */
   const { name, link } = req.body;
-  Card.create({ name, link, owner })
+  Card.create({ name, link })
```

Comment:
Нужно исправить: при создании карточки возвращается 500 и сообщение об ошибке.
Как воспроизвести:
1. Отправить `POST` запрос с заполненным `body` на `/cards`.
Что будет получено:
- в Postman:
```json
{
    "message": "card validation failed: owner: Path `owner` is required."
}
```

Тут осталась одна ошибка:
- не передается `owner` при создании записи (модель `Card` ожидает, что будет передан `id` пользователя, который хранится в `req.user._id`)
```suggestion
  Card.create({ name, link, owner: req.user._id })
```